content/project/board-election-process.adoc

```diff
@@ -13,153 +13,127 @@ This document outlines the link:/project/governance/#governance-board[Jenkins Go
+The candidates should also provide their statements, list of affiliations (see <<Corporate Disclosure>>), any account/social media links which may help voters to get more information about them, and plans if they are elected to the role.
 This information is published by the election committee before the voting starts.
 
-### Voter sign-up and eligibility
-
-Any Jenkins project and/or community contributor is eligible to vote in the election
-if there is a contribution made before September 01 of the election year.
-_Contribution_ does not mean a _code contribution_,
-all contributions count:
-documentation patches,
-code reviews,
-substantial issue reports,
-issues and mailing list responses,
-social media posts,
-testing,
-etc.
-Such a contribution should be public.
-
-Voter registration is announced through Jenkins mailing lists, blog, and social media accounts.
-Voters can register to vote in the election using one of two ways:
-
-1. Fill out the Google Form referenced in the announcement.
-2. Send a registration email to the election committee contact mailing list.
-
-In both cases, the following information should be provided during the sign up:
-
-* **Emails** that are required to send ballots through the link:https://civs.cs.cornell.edu/[Condorcet Internet Voting Service].
-These emails are deleted after the election and not used for any communications except the election matters.
-* **I agree with storing my email** -
-  Mandatory checkbox which gets consent for storing the email for the election process purposes.
-* **I agree with election terms** -
-  Mandatory checkbox which links this page and describes the key expectations
-  (e.g. voting only once) .
-* **Link to a contribution** -
-  Public link to a contribution that happened before Sep 01 of the election year.
-* **Jenkins account ID** (optional) -
-  User ID used to log into the Jenkins services like Jenkins Jira. 
-* **GitHub account ID** (optional) -
-  ID of the GitHub users for those who contribute through GitHub. 
-* **Contribution description** (optional) -
-  Free-form field which can be used to describe and justify the contribution(s) if the link is not enough.
-* Maybe: optional form entries selected by the election committee (e.g. election-related poll).
-* **Any feedback to the election committee** -
-  Additional entry where poll participants can provide any feedback.
+=== Voter sign-up and eligibility
+
+Any Jenkins project and/or community contributor is eligible to vote in the election if there is a contribution made before September 01 of the election year.
+Contributing does not only mean a _code contribution_, link:/participate[contributions] could be:
+
+* Documentation udpates or creation
+* Code reviews
+* Issue reporting
+* Translating resources
+* Connecting with or assisting the community
+* Code testing
+
+As long as you are contributing to the Jenkins project or community,  
```

Comment:
should there be more after this comma to complete the sentence?

Reply:
yup, forgot to finish that one, but have updated it now and will submit the update